resources/validations/validate_with_schematron.sh

```diff
@@ -0,0 +1,38 @@
+if [ ! -e  "$1" ]; then
+    echo "no file input for report, exiting"
+    exit 1
+fi
+DOC_TO_VALIDATE="$1"
+echo "doc requested to be validated: ${DOC_TO_VALIDATE}"
+
+# Delete pre-existing XSLT report
+rm -rf target/*.xsl;
+SAXON_VERSION=${SAXON_VERSION:-10.2}
+
+mvn -q org.apache.maven.plugins:maven-dependency-plugin:2.1:get \
+    -DrepoUrl=https://mvnrepository.com/ \
+    -DartifactId=Saxon-HE \
+    -DgroupId=net.sf.saxon \
+    -Dversion=${SAXON_VERSION}
```

Comment:
`shellcheck` recommends:

> [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086): Double quote to prevent globbing and word splitting.

```suggestion
    -Dversion="${SAXON_VERSION}"
```

Reply:
updated